pallets/metaverse/src/lib.rs

```diff
@@ -658,11 +663,10 @@ impl<T: Config> Pallet<T> {
 			TokenType::Transferable,
 			CollectionType::Collectable,
 			Perbill::from_percent(10u32),
-		);
-		// TO DO: Add class as metaverse parameter
+		)
```

Comment:
We need to make the owner of Land and Estate class is metaverse treasury account id, which only mint through code rather than metaverse owner.